cachito/nexus_scripts/configure_nexus.groovy

```diff
@@ -0,0 +1,139 @@
+// This script configures Nexus to be ready for Cachito and is idempotent
+import com.google.common.collect.Sets
```

Comment:
I can do that. I did intentionally create a lot of variables so that it was clear what was being input in the methods. I could just add comments instead.

Reply:
Having a lot of variables is not a bad thing. However, they all have the same scope. I could accidentally use reuse a variable later in the script. Reduce the scope of each variable, not the amount of variables.